yarn.lock

```diff
@@ -209,7 +209,7 @@
     "@babel/helper-explode-assignable-expression" "^7.10.4"
     "@babel/types" "^7.10.4"
 
-"@babel/helper-builder-react-jsx-experimental@^7.10.4", "@babel/helper-builder-react-jsx-experimental@^7.11.5":
```

Comment:
Can you remove this file by running `git checkout upstream/master -- yarn.lock`?

Reply:
Updated - thanks for the help